proxy/envoy/route.go

```diff
@@ -36,9 +36,22 @@ const (
 	OutboundClusterPrefix = "out."
 )
 
-// buildSSLContextWithSAN returns an SSLContextWithSAN struct with VerifySubjectAltName when auth is enabled.
-// Otherwise, it returns nil.
-func buildSSLContextWithSAN(hostname string, context *ProxyContext) *SSLContextWithSAN {
+// buildListenerSSLContext returns an SSLContext struct when auth is enabled.
+// Otherwise, it returns nil. This is used in listener SSLContext.
+func buildListenerSSLContext(mesh *MeshConfig) *SSLContext {
```

Comment:
@myidpt , I intend to call this function "buildListener(...)". But turns out the name is occupied in route.go. So I call it "buildListenerSSLContext(...)", and to make it consistent, I renamed the existing buildSSLContextWithSAN(...) to buildClusterSSLContext(...). Thanks.